ert_shared/tracker/evaluator.py

```diff
@@ -60,6 +63,11 @@ def _drain_monitor(self):
                         with self._state_mutex:
                             self._updates.append(event.data)
                             self._work_queue.put(None)
+                            if event.data.get("status") == "Cancelled":
+                                drainer_logger.debug(
+                                    "observed evaluation cancelled event, return"
+                                )
+                                return
                             if event.data.get("status") == "Stopped":
```

Comment:
Are these magic strings defined? Could you use them instead? I see `Stopped` got merged already, so that's unfortunate.

Reply:
I will use the ones from _ENSEMBLE_TYPE_EVENT_TO_STATUS

Reply:
Perhaps we could add a is_cancelled, is_stopped etc to the snapshot or would that violate its design?